src/types/helperTypes.d.ts

```diff
@@ -150,3 +150,5 @@ export type DataSourceForPlaceable<P extends PlaceableObject> = P extends Placea
     ? D['_source']
     : never
   : never;
+
+export type StatusEffect = ActiveEffectDataConstructorData & { id: string };
```

Comment:
This one doesn't really belong with the helper types, imo. I would probably move it either into the CONFIG, or to the `Token` class.

Looks good otherwise, though.

Reply:
moved the type to the token document